code/src/index.css

```diff
@@ -1,13 +1,188 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: #181818;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New", monospace;
 }
+
+.master-container {
+  width: 300px;
+  margin: auto;
+}
+
+@media (min-width: 768px) and (max-width: 1023px) {
+  .master-container {
+    width: 660px;
+  }
+}
+
+@media (min-width: 1024px) {
+  .master-container {
+    width: 864px;
+  }
+}
+
+.site-header {
+  color: #FFFFFF;
+  font-size: 20px;
+  margin-bottom: 6px;
+}
+
+@media (min-width: 768px) and (max-width: 1023px) {
+  .site-header {
+    padding: 0 15px;
+  }
+}
+
+@media (min-width: 1024px) {
+  .site-header {
+    font-size: 16px;
+    padding: 0 9px;
+    margin: 10px 0 5px 0;
+  }
+}
+
+.header-line {
+  height: 0.7px;
+  background-color: #383838;
+  margin-bottom: 16px;
+}
+
+@media (min-width: 768px) and (max-width: 1023px) {
+  .header-line {
+    margin: 0 15px 16px 15px;
+  }
+}
+
+@media (min-width: 1024px) {
+  .header-line {
+    max-width: 874px;
+    margin: 0 9px 16px 9px;
+  }
+}
+
+h2 {
+  font-family: Helvetica;
+  color: #ffffff;
+  font-size: 16px;
+  text-decoration: none;
+  margin-top: 6px;
+  margin-bottom: 0;
+}
+
+@media (min-width:1024px) {
+  h2 {
+    font-size: 14px;
+  }
+}
+
+a, a:visited {
+  font-family: Helvetica;
+  color: #ffffff;
+  font-size: 14px;
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration-color: #ffffff;
+  text-decoration: underline;
+}
+
+.artist-name {
+  font-family: Helvetica;
+  color: #a0a0a0;
+  font-size: 14px;
+}
+
+.artist-name:hover {
+  color: #ffffff;
```

Comment:
`color: #fff`

Reply:
:) I think CSS returns 6 letters by default, so we just let it :honeybee: